test/irb/test_ruby_lex.rb

```diff
@@ -65,10 +65,17 @@ def test_auto_indent
         Row.new(%q(    end), 4, 4),
         Row.new(%q(  end), 2, 2),
         Row.new(%q(end), 0, 0),
+        Row.new(%q([[[), nil, 6),
+        Row.new(%q(    ]), 4, 4),
+        Row.new(%q(  ]), 2, 2),
+        Row.new(%q(]), 0, 0),
```

Comment:
Could you add a new test method? I think the test for this commit should be separated from `test_auto_indent`, like `test_multiple_braces_in_a_line`.

```ruby
    def test_multiple_braces_in_a_line
      input_with_correct_indents = [
        Row.new(%q([[[), nil, 6),
        Row.new(%q(    ]), 4, 4),
        Row.new(%q(  ]), 2, 2),
        Row.new(%q(]), 0, 0),
        Row.new(%q([<<FOO]), nil, 0),
        Row.new(%q(hello), nil, 0),
        Row.new(%q(FOO), nil, 0),
      ]

      lines = []
      input_with_correct_indents.each do |row|
        lines << row.content
        assert_indenting(lines, row.current_line_spaces, false)
        assert_indenting(lines, row.new_line_spaces, true)
      end
    end
```

Reply:
Sounds good to me.